docs/sphinx/sections/api/apidocs/libraries/dagster-dbt.rst

```diff
@@ -52,6 +48,8 @@ Assets (dbt Core)
 
 .. autofunction:: get_asset_keys_by_output_name_for_source
 
+.. autoclass:: DbtProject
```

Comment:
Can we add some API examples + docstring to `DbtProject`? It looks barren rn: https://dagster-docs-qk9unul9n-elementl.vercel.app/_apidocs/libraries/dagster-dbt#dagster-dbt-project